src/devices/Adg731/samples/README.md

```diff
@@ -0,0 +1,102 @@
+# ADG731 - Samples
```

Comment:
I'd limit this file to purely code - how to use it and go through interesting scenarios and not debugging and setting up PI. For those instructions please add them to documentation folder somewhere in separate PR